content/en/docs/contribute/_index.md

```diff
@@ -34,29 +36,45 @@ To get involved with documentation:
 2. Familiarize yourself with the [documentation repository](https://github.com/kubernetes/website) and the website's [static site generator](https://gohugo.io).
 3. Make sure you understand the basic processes for [improving content](https://kubernetes.io/docs/contribute/start/#improve-existing-content) and [reviewing changes](https://kubernetes.io/docs/contribute/start/#review-docs-pull-requests).
 
-## Contributions best practices
+## Good first contributions for beginners
 
-- Do write clear and meaningful GIT commit messages.
-- Make sure to include _Github Special Keywords_ which references the issue and automatically closes the issue when PR is merged.
-- When you make a small change to a PR like fixing a typo, any style change, or changing grammar. Make sure you squash your commits so that you dont get a large number of commits for a relatively small change.
-- Make sure you include a nice PR description depicting the code you have changes, why to change a following piece of code and ensuring there is sufficient information for the reviewer to understand your PR.
-- Additional Readings : 
-    - [chris.beams.io/posts/git-commit/](https://chris.beams.io/posts/git-commit/)
-    - [github.com/blog/1506-closing-issues-via-pull-requests ](https://github.com/blog/1506-closing-issues-via-pull-requests )
-    - [davidwalsh.name/squash-commits-git ](https://davidwalsh.name/squash-commits-git )
+If you're new to Kubernetes, working in GitHub, or working on documentation, start here:
 
-## Other ways to contribute
+- [Suggest improvements](/docs/contribute/suggest-improvements/) to existing documentation and learn more about filing issues in GitHub.
+- Contribute a [small change](/docs/contribute/new-content/new-content/#small-changes) to existing content to learn the pull review process.
+- [Review pull requests](/docs/contribute/review/reviewing/) from other Kubernetes community members for accuracy and language.
```

Comment:
nit: The sub-header says, `Good first contributions for beginners`, do you need line 41?
nit: line 45, How about removing `for accuracy and language`. Hopefully the content in the reviewing page provides recommendations for how to review